### ISSUE TITLE
Crash when loading specific release

### ISSUE THREAD

**Issue by Nemris** [slapperwan/gh4a#1310]
OctoDroid crashes when attempting to load [ZFS](https://github.com/openzfs/zfs)' v2.2.0 release.

Crash log:
```
--------- beginning of crash
11-21 21:11:35.286 23162 23162 E AndroidRuntime: FATAL EXCEPTION: main
11-21 21:11:35.286 23162 23162 E AndroidRuntime: Process: com.gh4a, PID: 23162
11-21 21:11:35.286 23162 23162 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.gh4a/com.gh4a.activities.ReleaseInfoActivity}: java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary (2957 follows  )
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3846)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4022)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2336)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:246)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8653)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: Caused by: java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary (2957 follows  )
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:703)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:686)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HtmlUtils$HtmlToSpannedConverter.convert(HtmlUtils.java:383)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HtmlUtils$Html.fromHtml(HtmlUtils.java:264)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HtmlUtils.encode(HtmlUtils.java:238)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HttpImageGetter$ObjectInfo.encode(HttpImageGetter.java:198)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HttpImageGetter$ObjectInfo.bind(HttpImageGetter.java:171)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.utils.HttpImageGetter.bind(HttpImageGetter.java:393)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.activities.ReleaseInfoActivity.fillData(ReleaseInfoActivity.java:214)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.activities.ReleaseInfoActivity.handleReleaseReady(ReleaseInfoActivity.java:184)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at com.gh4a.activities.ReleaseInfoActivity.onCreate(ReleaseInfoActivity.java:92)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8207)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8191)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3819)
11-21 21:11:35.286 23162 23162 E AndroidRuntime: 	... 11 more
```

**Comment by sebastien46** [comment]
Reproducible on my device:
```
type: crash
osVersion: google/panther/panther:14/UQ1A.231205.015/2023121200:user/release-keys
package: com.gh4a:73
process: com.gh4a
processUptime: 55083 + 232 ms
installer: com.machiav3lli.fdroid

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.gh4a/com.gh4a.activities.ReleaseInfoActivity}: java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary (2957 follows  )
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3813)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3953)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2455)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8216)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
Caused by: java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary (2957 follows  )
	at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:694)
	at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:677)
	at com.gh4a.utils.HtmlUtils$HtmlToSpannedConverter.convert(HtmlUtils.java:383)
	at com.gh4a.utils.HtmlUtils$Html.fromHtml(HtmlUtils.java:264)
	at com.gh4a.utils.HtmlUtils.encode(HtmlUtils.java:238)
	at com.gh4a.utils.HttpImageGetter$ObjectInfo.encode(HttpImageGetter.java:198)
	at com.gh4a.utils.HttpImageGetter$ObjectInfo.bind(HttpImageGetter.java:171)
	at com.gh4a.utils.HttpImageGetter.bind(HttpImageGetter.java:393)
	at com.gh4a.activities.ReleaseInfoActivity.fillData(ReleaseInfoActivity.java:214)
	at com.gh4a.activities.ReleaseInfoActivity.handleReleaseReady(ReleaseInfoActivity.java:184)
	at com.gh4a.activities.ReleaseInfoActivity.onCreate(ReleaseInfoActivity.java:92)
	at android.app.Activity.performCreate(Activity.java:8677)
	at android.app.Activity.performCreate(Activity.java:8655)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1461)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3795)
	... 14 more

```

**Comment by Fs00** [comment]
This crash has been fixed in #1257, which is included in the latest version of the app.
